bridge/client/app/_utils/date.utils.ts

```diff
@@ -49,23 +52,46 @@ export class DateUtil {
     sameElse: string;
   } {
     if (showSeconds) {
-      return {
+      const calendarFormatsWithSeconds = {
         lastDay: '[yesterday at] HH:mm:ss',
         sameDay: '[today at] HH:mm:ss',
         nextDay: '[tomorrow at] HH:mm:ss',
         lastWeek: '[last] dddd [at] HH:mm:ss',
         nextWeek: 'dddd [at] HH:mm:ss',
         sameElse: 'YYYY-MM-DD HH:mm:ss',
       };
+      if (startUppercase) {
+        return {
+          ...calendarFormatsWithSeconds,
+          lastDay: '[Yesterday at] HH:mm:ss',
+          sameDay: '[Today at] HH:mm:ss',
+          nextDay: '[Tomorrow at] HH:mm:ss',
+          lastWeek: '[Last] dddd [at] HH:mm:ss',
+        };
+      }
+
+      return calendarFormatsWithSeconds;
     }
-    return {
+
+    const calendarFormats = {
```

Comment:
I don't really like the redundancy here with `showSeconds`. This could be simplified if we determine the time of the day.
```ts
const timeFormat = showSeconds ? 'HH:mm:ss' : 'HH:mm';
const calendarFormat = {
  lastDay: `[yesterday at] ${timeFormat}`,
  sameDay: `[today at] ${timeFormat}`,
  ...
};
```
Then we don't need to determine it again for `showSeconds = false`.

Reply:
Agreed. Thanks for your review! Here are my changes: https://github.com/keptn/keptn/pull/7283/commits/702f8549f9fd7b5430065f1d15629854d91b0a9b.